tests/kafkatest/services/kafka/config.py

```diff
@@ -28,7 +28,8 @@ class KafkaConfig(dict):
         config_property.METADATA_LOG_SEGMENT_BYTES: str(9*1024*1024), # 9 MB
         config_property.METADATA_LOG_BYTES_BETWEEN_SNAPSHOTS: str(10*1024*1024), # 10 MB
         config_property.METADATA_LOG_RETENTION_BYTES: str(10*1024*1024), # 10 MB
-        config_property.METADATA_LOG_SEGMENT_MS: str(1*60*1000) # one minute
+        config_property.METADATA_LOG_SEGMENT_MS: str(1*60*1000), # one minute
+        config_property.NEW_GROUP_COORDINATOR_ENABLE: False
```

Comment:
Nit: is there a better name than new group coordinator?  Not very descriptive and could easily become the old group coordinator if we arn't careful.